### DISCUSSION TITLE
Routing

### DISCUSSION THREAD

**Discussion by Dnkfff** [HowProgrammingWorks/LiveQA#165] [Q&A] [answered]
Зробіть, будь-ласка, лекцію чи репу по рутам адекватним на ноді, бо якщо писати в гуглі, routing node.js, там тільки що і діло, як один тільки експрес, є ще якісь альтернативи цій шайтан машини, бо наскільки знаю, всі зараз тільки і роблять, що використовують експрес у жсі :-)

**Comment by mykolachub** [comment, marked as answer]
Тимур Гафарович уже розповідав про сервер на ноді (routing, cluster, IP sticky)
Передивись: https://github.com/HowProgrammingWorks/NodeServer

**Comment by Dnkfff** [comment]
Дякую за відповідь!
Дивився її раніше, але знання завжди потребують оновлення, документація сильно з цим допомагає)
